smrf/tests/test_configurations.py

```diff
@@ -1,13 +0,0 @@
-from smrf.framework.model_framework import can_i_run_smrf
-from smrf.tests.smrf_test_case import SMRFTestCase
-
-
-class TestConfigurations(SMRFTestCase):
```

Comment:
Made the decision to remove this spec since I did not see much value. Other tests will start to fail and this one would not stand out.

Reply:
Never a fan of this test, it was always pretty pointless.